javascript/src/main/pages/Tutor/Tutor.js

```diff
@@ -6,22 +6,23 @@ import { fetchWithToken } from "main/utils/fetch";
 import { useAuth0 } from "@auth0/auth0-react";
 import Loading from "main/components/Loading/Loading";
 import TutorTable from "main/components/Tutor/TutorTable";
-import { useToasts } from "react-toast-notifications";
+//import { useToasts } from "react-toast-notifications";
 
 import {
-  buildCreateTutor,
-  buildDeleteTutor,
-  buildUpdateTutor
+  //buildCreateTutor,
+  buildDeleteTutor
+  //buildUpdateTutor
 } from "main/services/Tutor/TutorService";
 
 import { useHistory } from "react-router-dom";
 
 const Tutor = () => {
   const { user, getAccessTokenSilently: getToken } = useAuth0();
-  const { name, picture, email } = user;
+  //const { name, picture, email } = user;
+  const { email } = user;
   const history = useHistory();
   const { data: roleInfo } = useSWR(["/api/myRole", getToken], fetchWithToken);
-  const { addToast } = useToasts();
+  //const { addToast } = useToasts();
```

Comment:
Do we need these commented lines?

Reply:
looks like important cuz on tutors component but I'll take it out